src/server/http/__tests__/xsrf.js

```diff
@@ -29,10 +30,31 @@ describe('xsrf request filter', function () {
 
     await kbnServer.ready();
 
-    const routeMethods = nonDestructiveMethods.filter(method => method !== 'HEAD').concat(destructiveMethods);
```

Comment:
Okay it was a good "kibana debugging" excercise for me.

Since `_msearch` route is the proxy one, [payload parsing is disabled by default](https://github.com/hapijs/h2o2/blob/master/lib/index.js#L180-L185) and HapiJS API doc also mentions that payload.parse [defaults to true, __except when a proxy handler is used__](https://github.com/hapijs/hapi/blob/master/API.md). Disabled payload parsing allows us to pass any content-type without additional configuration, so here I can get rid of `config.payload.allow` and keep only `config.payload.parse = false`.